File: group1/JavaScript/models/project.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
   let Project = sequelize.define('Project',{
       title:{
           type: Sequelize.TEXT,
           allowNull: false,
       },
       description:{
           type: Sequelize.TEXT,
           allowNull: false,
       },
       budget:{
           type: Sequelize.INTEGER,
           allowNull: false,
       }
   },{
       timestamps:false
   });

    return Project;
};

File: group2/C#/ProjectRider/Models/Project.cs
namespace ProjectRider.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Project
    {
        [Key]
        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public int Budget { get; set; }

    }
}
